analysis/generate_report.py

```diff
@@ -97,6 +97,13 @@ def get_arg_parser():
               'get a report of all trials in "A" except for afl++ and all the '
               'trials from "B". "Later experiments" are those whose names come '
               'later when passed to this script.'))
+    mutually_exclusive_group.add_argument(
+        '-p',
+        '--merge-with-clobber-nonprivate',
+        action='store_true',
+        default=False,
+        help=('Does merge-with-clobber but includes all experiments that are '
```

Comment:
Can you expand more on description, unsure if someone will understand this completely.

Reply:
I don't want to repeat the help string for merge with clobber so I reference it explicitly now and tell people to read it.
And frankly, I don't expect anyone to use this option. It's mainly for me to debug what goes wrong with it since 
reporter.py will use this functionality.